### DISCUSSION TITLE
ScriptReference/GUILayout.TextArea

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#258] [Notes]
# ScriptReference/GUILayout.TextArea

https://docs.unity3d.com/ScriptReference/GUILayout.TextArea.html

**Comment by radiatoryang** [comment]
If you want cut/copy/paste support in editor, make sure you use `EditorGUILayout.TextArea` (warning: on these Editor classes, you can't access TextEditor without reflection!)
